docs/architecture/decisions/0025-caching

```diff
@@ -0,0 +1,71 @@
+# 24. Production Release Cadence
+
+Date: 2024-14-02
+
+## Status
+
+In Review
+
+## Context
+
+We experienced problems with our Cloudfront caching infrastructure early in our November launch.  In response, we turned off caching across the application.  We would like to utilize caching again without incurring the same issues.
+
+Details:
+Originally, Cloudfront was utilized to provide caching capabilities in our application.  All incoming HTTP requests first go through a Cloudfront endpoint, which has a caching infrastructure enabled by default. Cloudfront then decides whether to pass each request to our running Django app inside cloud.gov or if it will respond to with cached data.  The big problem with this feature is Cloudfront's caching has a default timeout of 24-hours, which we cannot control.  This led to issues on our November launch;  Incidents reported include the following...
```

Comment:
**Question:** 

1. Is it worth adding what caching capabilities we were using Cloudfront for?
2. Do we have error message that we saw that can be added here that led us to realising that Cloudfront was the culprit?

Reply:
I think the diagnosis was pretty involved.  I don't have any documentation on it.  My guess is maybe Neil would have something, but so far I'm just going to leave it and wait to see if anyone else can offer some snippets...?